com.unity.testing.urp/Scripts/Editor/MultipleViewGCTest.cs

```diff
@@ -44,7 +44,7 @@ public void RenderSceneAndGameView()
 
         if (allocationCountOfRenderPipeline > 0)
         {
-            Debug.LogError($"Memory was allocated {allocationCountOfRenderPipeline} times");
+            UnityEngine.TestTools.LogAssert.Expect(LogType.Error, $"Memory was allocated {allocationCountOfRenderPipeline} times");
```

Comment:
Why are we changing from logging an error to telling the test frame work to expect an error?